MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -9,38 +9,20 @@ import UIKit
 #endif
 
 /**
- `NavigationLocationManager` is the base location manager which handles
- permissions and background modes.
+ `NavigationLocationManager` is the base location manager which handles permissions and background modes.
  */
 @objc(MBNavigationLocationManager)
 open class NavigationLocationManager: CLLocationManager {
     
     var lastKnownLocation: CLLocation?
     
-    /**
-     Indicates whether the location manager’s desired accuracy should update
-     when the battery state changes.
-     */
-    @available(*, deprecated, message: "Manually override desiredAccuracy instead.")
-    public var automaticallyUpdatesDesiredAccuracy = true
-    
-    var accuracyOverride: CLLocationAccuracy?
-    
-    override open var desiredAccuracy: CLLocationAccuracy {
-        get {
-            if let override = accuracyOverride { return override }
-            return  UIDevice.current.isPluggedIn ? kCLLocationAccuracyBestForNavigation : kCLLocationAccuracyBest
-        }
-        set {
-            accuracyOverride = newValue
-        }
-    }
-    
     override public init() {
         super.init()
         
         requestWhenInUseAuthorization()
         
+        desiredAccuracy = kCLLocationAccuracyBest
```

Comment:
This is the default value on iPhones, right? Do we (or can we) force the value to be `kCLLocationAccuracyBest` on iPads?

Reply:
I think it would make me sleep better at night knowing we have something set here.

Reply:
Instead of a no-op in the code, let’s add a test that verifies the default value.